### ISSUE TITLE
Add support for multiple speakers

### ISSUE THREAD

**Issue by dadoonet** [brownbaglunch/bblfr_data#176]
Follow up as per #175 

We should have a list of associated speakers may be. Or an independant list of speakers and an independant list of talks. In the list of talks we can reference one or more speakers.
May be not ideal for PRs though.

**Comment by sebastienfr** [comment]
Good idea, I have also a talk with a co-speaker, and for now we should duplicate some content.
